pacman/riscv64.patch

```diff
@@ -25,14 +26,6 @@ index 5321ff1b..b20f85ee 100644
  
  
  prepare() {
-@@ -43,6 +45,7 @@ prepare() {
-   patch -Np1 -i ../makepkg-use-ffile-prefix-map-instead-of-fdebug-prefi.patch
-   patch -Np1 -i ../libmakepkg-add-extra-buildflags-only-when-buildflags.patch
-   patch -Np1 -i ../make-link-time-optimization-flags-configurable.patch
-+  patch -Np0 -i ../makepkg-config.guess.patch
```

Comment:
If we are going to remove this hack altogether, we should also remove the patch itself and from the source array.